version.go

```diff
@@ -275,16 +277,16 @@ func (v Version) bigger(v2 Version) bool {
 }
 
 // isAlphaNumeric reports whether c is an alpha character or digit
-func isAlphaNumeric(c uint8) bool {
+func isAlphaNumeric(c rune) bool {
 	return isDigit(c) || isAlpha(c)
 }
 
 // isAlpha reports whether c is an alpha character
-func isAlpha(c uint8) bool {
-	return 'a' <= c && c <= 'z' || 'A' <= c && c <= 'Z'
+func isAlpha(c rune) bool {
+	return unicode.IsLetter(c)
 }
 
 // isDigit reports whether d is an ASCII digit
-func isDigit(d uint8) bool {
-	return '0' <= d && d <= '9'
+func isDigit(d rune) bool {
+	return unicode.IsDigit(d)
```

Comment:
The changes to `isAlpha` and `isDigit` are the key changes of this PR. Instead of being restricted to the ASCII characters, all Unicode characters of those categories are considered valid.

Reply:
I see the `unicode.IsLetter` now. So I assume it will work fine. Just thought I'd ask.